docs/cep/windows/window-types/sliding-length.md

```diff
@@ -0,0 +1,32 @@
+---
+title: WINDOW SLIDING_LENGTH()
+---
+
+A _sliding length window_ that holds the last `length` events at a given time, and gets updated for each arrival and expiration.
+
+## Syntax
+
+    WINDOW SLIDING_LENGTH(length <INT>)
+
+## Query Parameters
+
+| Name          | Description     | Default Value | Possible Data Types | Optional | Dynamic |
+|----------|---------------------------------------|-------------|-----------------|----------|---------|
+| length | The number of events that should be included in a sliding length window. |          | INT        | No       | No      |
+
+## Example
+
+```sql
+CREATE WINDOW StockEventWindow (symbol string, price float, volume int) SLIDING_LENGTH(10) output all events;
```

Comment:
@koshy-thomas or @StoyanTc, apparently this needs one or more streams to work. Can you fix this so it is valid?